added more instructions to create pipeline (#237)
AICoE/elyra-aidevsecops-tutorial: docs/source/create-ai-pipeline.md

## Add runtime images (using UI)

> Now that your images are available on the registry, we need to add them to [Elyra][1] metadata. For that, we need to create a runtime image which identifies the container image that Elyra can utilize to run Jupyter notebooks. This container image contains the dependencies required for running the notebooks.

Author: yes, we are adding the uri of the notebook images that we created in the previous steps by creating a "runtime image" as they call it, thats how it knows where to find the container image https://elyra.readthedocs.io/en/latest/user_guide/runtime-image-conf.html#runtime-image-configuration

the next step is creating the kubeflow pipeline runtime configuration which is different from this.

> Now that your images are available on the registry, we need to add them to [Elyra][1] metadata. For that, we need to create a runtime image which identifies the container image that Elyra can utilize to run Jupyter notebooks. This container image contains the dependencies required for running the notebooks.

Reviewer: according to this link I think they call it a "runtime image configuration", and not a "runtime image". I think this edit might make the most sense. WDYT? 

```suggestion
Now that your images are available on the registry, we need to add them to [Elyra][1] metadata. For that, we need to create a runtime image configuration which identifies the container image that Elyra can pull from our registry and utilize to run the Jupyter notebooks.
```

> Now that your images are available on the registry, we need to add them to [Elyra][1] metadata. For that, we need to create a runtime image which identifies the container image that Elyra can utilize to run Jupyter notebooks. This container image contains the dependencies required for running the notebooks.

Author: made the changes :)

### Create runtime to be used in Kubeflow pipeline (using UI)

> We also need to create a Kubeflow Pipeline Runtime configuration which provides Elyra access to the external Kubeflow Pipeline where we wish to run the pipeline that we are creating.

Author: done :)

- Filename: This is the name of the node. this would be pre-filled when selecting a script or a notebook's properties.
- Runtime Image: Select the runtime image that was created in the previous steps. This identifies the container image required to run this node.
- CPU, GPU, RAM: Specify the resource requirements needed to run this script or notebook.
- File Dependencies: Any files that the notebook is dependent on (notebooks and scripts) such as for importing methods from can be specified here.
- Environment Variables: Specify the environment variables which are utilized within the notebooks and scripts here.
- Output Files: You can specify any files which are created during the execution and which might be needed by subsequent steps here.

<div style="text-align:center">
<img alt="Pipeline Steps Inputs" src="https://raw.githubusercontent.com/thoth-station/elyra-aidevsecops-tutorial/master/docs/images/AIPipelineStepInputs.png">
</div>

The notebooks can exchange data using a cloud object storage and also using the Output Files which can be used to make certain files generated during execution available to all subsequent pipeline steps.

> The `download_dataset` and `training` step of the pipeline have some mandatory and optional environment variables for running in automation which need to be set during this step of setting up the pipeline.

Reviewer 2: Are we able to have a screenshot of the panel with all env variables added or at least an explicit list of what environment variables you must add for the pipeline to run? Reading through this doc, there's lots of good suggestions of what to add, but its not clear what the minimal requirements are.

Author: @isabelizimm thanks for pointing that out! in the section just below this we have a minimal list of variables that need to be configured. It was called "NOTE on notebook properties" but I now made it a little more clear. And added a couple images which show those environment variables.
